fuzzing/tools/launcher.py

```diff
@@ -11,23 +26,31 @@
 FLAGS = flags.FLAGS
 
 flags.DEFINE_integer(
-    'timeout_secs',
+    "timeout_secs",
     20,
-    'The maximum duration, in seconds, of the fuzzer run launched.',
+    "The maximum duration, in seconds, of the fuzzer run launched.",
     lower_bound=0)
 
+flags.DEFINE_string("corpus_dir", "", "The directory containing corpus.")
+
 
 def main(argv):
     if len(argv) != 2:
         raise app.UsageError(
             "This script receives 1 argument. It should look like:" +
             "\n\tpython " + __file__ + " EXECUTABLE")
 
-    os.execv(argv[1], [
-        argv[1], "-max_total_time=" + str(FLAGS.timeout_secs),
-        "-timeout=" + str(FLAGS.timeout_secs)
-    ])
+    if FLAGS.corpus_dir:
+        os.execv(argv[1], [
+            argv[1], FLAGS.corpus_dir, "-max_total_time=" +
+            str(FLAGS.timeout_secs), "-timeout=" + str(FLAGS.timeout_secs)
+        ])
+    else:
+        os.execv(argv[1], [
+            argv[1], "-max_total_time=" + str(FLAGS.timeout_secs),
+            "-timeout=" + str(FLAGS.timeout_secs)
+        ])
```

Comment:
Same comment about duplication as above. Consider instead constructing the argument list separately, and using conditionals to append arguments to the list. Something like this:

```py
command_line = [argv[1]]
command_line.append("-max_total_time=" + str(FLAGS.timeout_secs))
command_line.append("-timeout=" + str(FLAGS.timeout_secs))
if FLAGS.corpus_dir:
  command_line.append(...)
os.execv(argv[1], command_line)
```